Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -1019,14 +1019,13 @@ Ref<API::PageConfiguration> WebPageProxy::protectedConfiguration() const
     return m_configuration;
 }
 
-
 ProcessID WebPageProxy::gpuProcessID() const
 {
     if (m_isClosed)
         return 0;
 
 #if ENABLE(GPU_PROCESS)
-    if (auto* gpuProcess = configuration().processPool().gpuProcess())
+    if (RefPtr gpuProcess = protectedConfiguration()->processPool().gpuProcess())
```

Comment:
Can't we inline `processPool()` to avoid having to call `protectedConfiguration()`?

Reply:
I just tried this--we can't inline it. The header APIPageConfiguration.h needs to know about the WebProcessPool type to use LazyInitializedRef properly :(